README.md

```diff
@@ -55,7 +55,7 @@ jobs:
   bump:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```

Comment:
Looking at https://github.com/actions/checkout they seem to use a version (`@v4`) maybe should switch to that?

Reply:
Good point, yeah :)